README.md

```diff
@@ -1,2 +1,2 @@
 # piisa
-Personal information identification standard
+This is the Personal information identification standard (Piisa) repository. We are working on an on open specifications for interoperating between variouos privacy API frameworks. 
```

Comment:
there is a typo here: _we are working on an **on** open_

Reply:
good catch